ui/guest/src/components/GuestProxy.js

```diff
@@ -277,19 +284,39 @@ export function GuestProxy(props) {
             $clone.insertBefore($siblings.eq(targetIndex));
           }
 
-          forEach(
-            $daddy.children(),
-            (el, i) => {
+          registerDeregisterElements(Array.from($daddy.children()), 'insert', targetIndex);
```

Comment:
I think jQuery does give you an array, no?

Reply:
it gives a htmlCollection and it does not have forEach